### PR TITLE
DO NOT SUBMIT: Removed experiment flags from runtime methods for Update Subscription

### PR DESCRIPTION
**Pull request by Kestrad**

### CODE REVIEW

File: src/runtime/runtime.js [hunk 1]
```diff
@@ -775,10 +775,6 @@ export class ConfiguredRuntime {
 
   /** @override */
   showUpdateOffers(options) {
-    assert(
```
Comment (alin04) [inline src/runtime/runtime.js]: Should we ramp the experiment to 100% first? Or instruct the publisher to override with query param (if all they're doing is testing)?

Comment (Kestrad) [reply src/runtime/runtime.js]: @jpettitt I think?

Comment (Kestrad) [reply src/runtime/runtime.js]: After talking to Angie and John we've decided to ramp the experiment instead. Keeping this PR open for the time being, but not submitting.

**Comment by ChrisAntaki** [comment]
Thanks for your work on this, @Kestrad !

**Pull request closed without merging**